### PR TITLE
fix(validator): throw error if getMessage not of type Function

### PR DESCRIPTION
**Pull request by T-Motion**
#544 fix

### CODE REVIEW

File: packages/validate/test/Validator.test.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
 import { expect, fixture, html, unsafeStatic, defineCE } from '@open-wc/testing';
-import { LitElement } from '@lion/core';
 import sinon from 'sinon';
+import { LitElement } from '../../core/index.js';
```
Comment (daKmoR) [inline packages/validate/test/Validator.test.js]: that needs to stay `@lion/core` otherwise it only works within the monorepo and not for our users 🙈

Comment (T-Motion) [reply packages/validate/test/Validator.test.js]: Ofcourse, new commit pushed 😊

Comment (daKmoR) [reply packages/validate/test/Validator.test.js]: tests are not real user code - still, from an npm package point of view it is an "external" package so let's be consistent 👍

Comment (daKmoR) [reply packages/validate/test/Validator.test.js]: nice 👌

Comment (T-Motion) [reply packages/validate/test/Validator.test.js]: Yes ofcourse, i agree. I updated it because i first did "npm i" instead of "yarn install" 😊

**Comment by daKmoR** [comment]
fyi this particular test is flaky so the ci is fine 👍

**Comment by daKmoR** [comment]
nice 👌  this will help a lot of people using it correctly 💪

**Pull request merged**